docs/products/postgresql/concepts/high-availability.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: High availability
+title: High availability of Aiven for PostgreSQL®
+sidebar_label: High availability
 ---
 
-Aiven for PostgreSQL® is available on a variety of plans, offering
-different levels of high availability. The selected plan defines the
-features available, and a summary is provided in the table below:
+Aiven for PostgreSQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available, and a summary is provided in the table below:
 
- | Plan         | High Availability Features                                                     | Backup History |
- | ------------ | ------------------------------------------------------------------------------ | -------------- |
- | **Hobbyist** | Single-node with limited availability                                          | 2 days         |
- | **Startup**  | Single-node with limited availability                                          | 2 days         |
- | **Business** | Two-node (primary + standby) with higher availability                          | 14 days        |
- | **Premium**  | Three-node (primary + standby + standby) with top availability characteristics | 30-day         |
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: basic high availability)          | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: stronger high availability) | 30-day         | 99.99 % |
 
 ## About primary and standby nodes
 
 Aiven's Business and Premium plans offer
 [primary nodes](/docs/products/postgresql/reference/terminology) and
```

Comment:
I think the confusion around primaries comes from this original line. There is always one primary and 1/2 standby(s) in our plans. I think the original wording was confusing already as it reads like "multiple primaries" in a cluster which is incorrect. Otherwise it would be a multi-master/multi-primary solution.

I think the correct one-liner expression should be "Aiven's Business and Premium plans offer a primary node and standby nodes." Even this would still be ambigious because Business plans have only one standby node whereas Premium plans have 2.

Please also refer to my other comment for "primaries" below.